### ISSUE TITLE
FAB displayed on wrong tab

### ISSUE THREAD

**Issue by adneal** [UweTrottmann/SeriesGuide#476]
It's possible for the FAB to be displayed on the "now" tab as a result of the `FabAbsListViewScrollDetector` getting confused about which `Fragment` is currently being shown. Personally, I don't like hiding the FAB, but I'm not sure how you'd like to handle it. An easy fix would be just to check to see if the grid has focus or not:

        if (!view.hasFocus() || totalItemCount == 0) {
            return;
        }

Also, what are you thoughts on switching to an actual `Snackbar` instead of your custom one? Is there a reason for using it?

![device-2015-10-07-193743](https://cloud.githubusercontent.com/assets/489364/10354910/ff4d5a8e-6d2a-11e5-8018-2837f36c2907.png)

**Comment by UweTrottmann** [comment]
I tried leaving it visible all the time, but a lot of users did not like this (especially on small screens, it takes away a lot of space). Thanks for the tip, will try this fix (have not come up with a good solution myself so far)!

As for the custom snackbar: I tried using the design library one. But it attaches to a view outside the fragment, so it would be visible on all tabs. I though about some workarounds, but for now it was easier to keep the inline, custom one around.
